.github/workflows/ci.yaml

```diff
@@ -653,3 +640,25 @@ jobs:
           [ "${{matrix.settings.name}}" = "Sqlserver" ] && exit 0
 
           just slt-bin --protocol=flightsql --exclude '*/tunnels/ssh' ${{matrix.settings.path}}
+
+  docker-push:
+    name: Build and Push Docker Image
+    if: github.event_name == 'push' && github.repository == 'GlareDB/glaredb' && github.ref == 'refs/heads/main'
```

Comment:
the repository check probably doesn't matter, but it's fine.

Reply:
we should, however, make this task depend on... some stuff.

Reply:
> the repository check probably doesn't matter, but it's fine.

True it would fail in forks, just thought it didn't hurt.